cove/cove_360/tests_functional.py

```diff
@@ -170,6 +170,12 @@ def server_url(request, live_server):
     ("dei_extension.xlsx", [
         "do not use the 360Giving Data Standard codelists correctly.",
     ], True),
+    ("ids-strange-chars.xlsx", [
+        "2 grants have a Funding or Recipient Organisation identifier contains unexpected characters",
+    ], True),
+    ("ids-strange-chars.xlsx", [
+        "a Grant Identifier contains unexpected characters."
```

Comment:
```suggestion
        "2 grants have a Funding or Recipient Organisation identifier containing unexpected characters",
    ], True),
    ("ids-strange-chars.xlsx", [
        "a Grant Identifier containing unexpected characters."
```

(see other suggested changes)

Reply:
Also should that be "1 grant has a Grant Identifier ..." ?

Reply:
The formatting for plurals does "a" instead of a number if it's singular so I think this is correct.

Reply:
fixed in later pr